main/src/cgeo/geocaching/connector/al/ALApi.java

```diff
@@ -98,28 +181,28 @@ protected static Collection<Geocache> searchByBBox(final Viewport viewport) {
         Log.d("_AL Radius: " + radius);
 
         final Geopoint center = new Geopoint(latcenter, loncenter);
-        return searchByCenter(center, radius);
+        return searchByCenter(center, radius, null);
     }
 
     @NonNull
     protected static Collection<Geocache> searchByCenter(final Geopoint center) {
-        return searchByCenter(center, DEFAULT_RADIUS);
+        return searchByCenter(center, DEFAULT_RADIUS, null);
     }
 
     @NonNull
     @WorkerThread
-    private static Collection<Geocache> searchByCenter(final Geopoint center, final int distanceInMeters) {
+    private static Collection<Geocache> searchByCenter(final Geopoint center, final int distanceInMeters, final Integer daysSincePublish) {
```

Comment:
It somehow seems weird to me that a method called "searchByCenter" should take a parameter "daysSincePublish". Isn't it possible to somehow replace this function completely with calls to "searchByFilter"?

Reply:
searchByCenter seems to be a common thing in the codebase, as opposed to searchByBBox. Didn't want to touch that.